.github/workflows/build-yocto.yml

```diff
@@ -14,10 +14,19 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Run kas lock
```

Comment:
You should do the kas lock / upload in a dedicated job, not in the yocto check layer. so that we can then (independently) decide if the 'compile' jobs depend or not on the yocto check layer.

Reply:
added a new job to create the lock. thanks